### PR TITLE
Migrate to commercetools conventions

### PR DESCRIPTION
**Pull request by hisabimbola**
Fixes #8

### CODE REVIEW

**Comment by Siilwyn** [comment]
:bowtie: Looking good, two minor things:
- Make `npm test` run the linter since it's the only test we have. [L49](https://github.com/commercetools/eslint-config/blob/migrate-to-commercetools-conventions/package.json#L49)
- The `.eslintrc` file should have the [file format specified](http://eslint.org/docs/user-guide/configuring#configuration-file-formats).

**Comment by hisabimbola** [comment]
@Siilwyn testing is different from linting.

**Comment by Siilwyn** [comment]
Well it *tests* if your code adheres to the rules you set. Also I'd rather have a double command than no `test` command at all.

**Comment by hisabimbola** [comment]
@PhilippSpo you good with this?

File: README.md [hunk 1]
```diff
@@ -1,13 +1,34 @@
+![commercetools-icon](https://github.com/commercetools/press-kit/blob/master/PNG/72DPI/CT%20logo%20horizontal%20RGB%2072dpi.png?raw=true)
+
 # eslint-config-commercetools
 
+[![David Dependencies Status][david-icon]][david]
+[![Travis Build Status][travis-icon]][travis]
```
Comment (PhilippSpo) [inline README.md]: Do you want to have a travis build for this lib?

Comment (hisabimbola) [inline README.md]: Good question, not sure what the build will do though 💭 . I will remove it for now and add it back if need be

Comment (PhilippSpo) [inline README.md]: 👍

Comment (hisabimbola) [inline README.md]: Updated

**Comment by PhilippSpo** [comment]
LGTM 👍

**Pull request merged**